pages/task/read/views/pil.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Fragment } from 'react';
+import moment from 'moment';
+import { Snippet } from '@asl/components';
+import { dateFormat, procedureDefinitions } from '../../../../constants';
+import { connect } from 'react-redux';
+
+const Pil = ({ profile }) => {
+  const pil = profile.pil;
+  return (
+    <Fragment>
+      <h2><Snippet>pil.training.title</Snippet></h2>
+      { profile.certificates.length && profile.certificates.map((certificate, index) => (
+        <div className="certificate" key={index}>
```

Comment:
we are using a className here and an id below for 'exemptions' and 'procedures'. My question though is do we need either? Are they actually used for anything?

Reply:
I ask as I can't see any styles being added or these being referenced anywhere so seems a bit like clutter.

Reply:
It _might_ be that it's worth adding some references to assist in integration testing, but 100% agree that if that's what we're doing then they should be consistent (and possibly use a different attribute other than `className`- i.e. `data-*` -  to disambiguate from classes applied to styling)

Reply:
That is right, we do not need the classNames, I removed them. We need the ids to be able to [navigate](https://github.com/UKHomeOffice/asl-pages/blob/439b46d5f065aa2be090059c85f8db80c4a68d1a/pages/task/read/views/index.jsx#L72).